docs/source/sysadmin_guide/testing_sawtooth.rst

```diff
@@ -32,6 +32,49 @@ to test basic Sawtooth functionality.
    If not, check the status of the REST API service and restart it, if
    necessary; see :doc:`systemd`.
 
+#. If this node has joined an existing network, use the following steps to
+   confirm network functionality.
+
+   a. To check whether peering has occurred on the network, submit a peers query
+      to the REST API on this node.
+
+      .. code-block:: console
+
+         $ curl http://localhost:8008/peers
+
+      .. note::
+
+         If this node is not running a local REST API, replace
+         ``localhost:8008`` with the externally advertised IP address and port
+         of the REST API.
+
+      You should see a JSON response that includes the IP address and port for
+      the validator and REST API, as in this example:
+
+      .. code-block:: console
+
+         {
+             "data": [
+             "tcp://validator-1:8800",
+           ],
+           "link": "http://rest-api:8008/peers"
+         }
```

Comment:
I think the whitespace/tabbing is messed up above, should be:
```
         {
           "data": [
             "tcp://validator-1:8800",
           ],
           "link": "http://rest-api:8008/peers"
         }
```
(data has two extra spaces)

Reply:
The source file doesn't have any extra spaces before "data".  I hope Jenkins isn't inserting spurious indents in the build artifacts.